hack/cluster-build.sh

```diff
@@ -37,4 +37,30 @@ ${KUBEVIRT_PATH}hack/dockerized "DOCKER_PREFIX=${DOCKER_PREFIX} DOCKER_TAG=${DOC
 ${KUBEVIRT_PATH}hack/dockerized "DOCKER_PREFIX=${DOCKER_PREFIX} DOCKER_TAG=${DOCKER_TAG} KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER} ./hack/bazel-build.sh"
 ${KUBEVIRT_PATH}hack/dockerized "DOCKER_PREFIX=${DOCKER_PREFIX} DOCKER_TAG=${DOCKER_TAG} DOCKER_TAG_ALT=${DOCKER_TAG_ALT} KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER} ./hack/bazel-push-images.sh"
 
+# Make sure that all nodes use the newest images
+container=""
+container_alias=""
+for arg in ${docker_images}; do
+    name=$(basename $arg)
+    container="${container} ${manifest_docker_prefix}/${name}:${docker_tag} ${manifest_docker_prefix}/${name}:${docker_tag_alt}"
+    container_alias="${container_alias} ${manifest_docker_prefix}/${name}:${docker_tag} kubevirt/${name}:${docker_tag}"
+done
+
+# OKD provider has different node names and does not have docker
+if [[ $KUBEVIRT_PROVIDER =~ okd.* ]]; then
```

Comment:
the crio ocp3 lane is podman as well.

Reply:
os-3.11-crio still uses the docker to work with images so it should be fine